cpp/marc_tools/marc_grep_multiple.sh

```diff
@@ -22,25 +22,30 @@ for filename in "$@"; do
             found=0
 	fi
     else
-	tar_filename=${filename%.gz}
+    filebasename="/tmp/$(basename ${filename})"
+	tar_foldername=${filebasename%.gz}
```

Comment:
strange indentation in this file, tabs and spaces seem to be mixed. please normalize!